docs/release_guide.md

```diff
@@ -1,43 +1,43 @@
-7) Push your branch to remote, and run `./Utils/gitlab_triggers/trigger_content_nightly_build.sh -ct <GitLab_token> -b <new_content_branch_name>`.
-  **Note:** if you're on `content/master`, a notification will be sent to the content-team slack channel. The destination channel can be set via argument.
-  Wait until the nightly sdk completes (around 2-3h, mostly for validation)
-7) Open a PR for that content branch, and verify that the build triggered is green. Note that in order to trigger the build, opening the PR is required.
-8) Enter the content-internal-dist repo, open a new branch and update the demisto-sdk version in the [**.gitlab-ci.yml**](https://code.pan.run/xsoar/content-internal-dist/-/blob/master/.gitlab/.gitlab-ci.yml) file. Use the release branch - replace the `pip3 install git+https://github.com/demisto/demisto-sdk.git@master#egg=demisto-sdk` line with this line: `pip3 install git+https://github.com/demisto/demisto-sdk.git@release-branch-name#egg=demisto-sdk.`
-9) Push your branch to remote, and run `./.gitlab/trigger_content_gold_nightly_build.sh -ct <GitLab_token> -b <new_internal_dist_branch_name>`.
-  **Note:** if you're on `content-internal-dist/master`, a notification will be sent to the content-team slack channel. The destination channel can be set via argument.
-  Wait until the nightly sdk completes.
-  **Note:** you should discard this change and delete the branch after the build is finished successfully.
-10) In Demisto's content-private repository create a new branch and go to the build configuration file [**config.yml**](https://github.com/demisto/content-private/blob/master/.github/workflows/config.yml) and update the SDK installation command by replacing `pip3 install demisto-sdk` with `pip3 install git+https://github.com/demisto/demisto-sdk.git@<sdk-release-branch-name>.`
-11) Open a PR for that content-private branch, and verify the build triggered is green. Note that **this PR is for SDK version check only, and it shouldn't be merged**, once the build is successful, discard this change and close the PR.
-
-**Note:** Steps 6, 7, 9, 11 can be performed at the same time (no need to wait for one to finish before starting the other).
+Throughout the guide, `X.X.X` will mark the upcoming version.
+
+## Pre-release validations 
+
+1) Branch out from the latest SDK PR used for a nightly test, name the branch `X.X.X`. 
+2) Make sure the **CHANGELOG.md** file is in order. Add a subtitle `## X.X.X`.
+3) Run `poetry update` (in the SDK folder).
+5) Run `poetry version <new version number>`.
+6) Make sure that all the following passed:
+  - [content nightly](https://code.pan.run/xsoar/content/-/pipeline_schedules)
```

Comment:
Maybe also add here that it runs against latest sdk, and that sdk master runs against content? To reduce confusion.